GVRf/Framework/framework/build.gradle

```diff
@@ -73,7 +76,7 @@ android {
     }
 
     sourceSets.main {
-        java.srcDirs = ['src/main/java', 'src/main/backends/oculus']
+        java.srcDirs = ['src/main/java', '../' + rootProject.property("GVRF_BACKEND")+'/src/main/java']

```

Comment:
Also if Google VR is backend 2, we would need a way to compile both backend 1 an 2 and load backend 2 if backend 1 is not available.